src/parse/app.rs

```diff
@@ -1,27 +1,43 @@
 use std::collections::HashSet;
 
-use indexmap::map::Entry;
+// use indexmap::map::Entry;
 use proc_macro2::TokenStream as TokenStream2;
 use syn::{
     parse::{self, ParseStream, Parser},
     spanned::Spanned,
-    ExprParen, Fields, ForeignItem, Ident, Item, Lit, Path, Token, Visibility,
+    Expr,
+    ExprArray,
+    ExprParen,
+    Fields,
+    ForeignItem,
+    Ident,
+    Item,
+    //Lit,
```

Comment:
When we're satisfied with the functionality the unused imports could be removed

Reply:
Fixed along with other stuff.